tailwind.config.ts

```diff
@@ -23,6 +23,7 @@ module.exports = {
       borderRadius: {
         xs: '0.125rem', // 2px
         sm: '0.25rem', // 4px
+        full: '9999px',
```

Comment:
In design, 50px was the largest radius for now, but 9999px is better one for oval shaped UI components.
In default tailwind, 9999px is named as "full" .

Reply:
Yeah you're right, we didn't need to explicitly define rounded-full...
I'll remove it!